Felienne/hedy
# [FEATURE] Enables teachers to toggle developers mode and cheatsheet in customization

**Description**
In this PR we add a small new feature that was already supported before the class customizations re-structuring. We re-implement the feature to enable teachers to enforce the developer's mode and toggle the visibility of the _cheatsheet_. This can be useful for an exam or to verify the students programming abilities as they are no longer able to get hints from the adventures or cheatsheet. It also increases the size of the  editor making it easier to create large programs.

**Fix for**
This PR fixes #1975 and fixed #2012.

**How to test**
Navigate to your class and notice the new UI of the customizations. Select the "mandatory developer's mode" and/or the "hide cheatsheet" and save the customizations. Verify that a student can only see the developers mode or cheatsheet when the corresponding setting is enabled.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
